scripts/src/java/org/oppia/android/scripts/coverage/RunCoverageForTestTarget.kt

```diff
@@ -65,7 +66,7 @@ class RunCoverageForTestTarget(
   /**
    * Analyzes target file for coverage, generates chosen reports accordingly.
    */
-  fun runCoverage(): List<String>? {
+  fun runCoverage(): CoverageReport? {
```

Comment:
Here & elsewhere: we probably should throw an exception rather than return null if we can produce a report (especially since that should only happen in exceptional cases).

Reply:
Would this also apply to the CoverageRunners's retrieveCoverageResult that retrieves the result from bazelClient?
```
  private fun retrieveCoverageResult(
    bazelTestTarget: String
  ): List<String>? {
    return bazelClient.runCoverageForTestTarget(bazelTestTarget)
  }
```

Reply:
Now it uses - `error("Failed to retrieve coverage result for $bazelTestTarget")`

Reply:
This file is nolonger in this PR. Did you move it to a different one?